pkg/common/password/manager.go

```diff
@@ -19,3 +22,12 @@ func (m SimpleManager) GeneratePassword(length int) (string, error) {
 	}
 	return hex.EncodeToString(bytes), nil
 }
+
+func (m SimpleManager) GenerateHtpasswd(username string, password string) (string, error) {
+	htpasswd, err := exec.Command("htpasswd", "-bn", username, password).Output()
+	if err != nil {
+		return "", err
+	}
+	formattedHtpasswd := strings.TrimSpace(string(htpasswd))
+	return formattedHtpasswd, nil
```

Comment:
We can probably get rid of the `htpasswd` dependency and use something like


```suggestion
	encryptedPassword, err := bcrypt.GenerateFromPassword([]byte(password), bcrypt.DefaultCost)
	if err != nil {
		return "", err
	}
	formattedHtpasswd := fmt.Sprintf("%s:%s", username, string(encryptedPassword))
	return formattedHtpasswd, nil
```

Where `bcrypt` is `"golang.org/x/crypto/bcrypt"`

Need to be tested

Reply:
I'm gonna try this without the username being added as well so that it stays compliant with the rest of the code

Reply:
Beware that `htpasswd` does add the username in front of the hashed password, producing a string in the following format:
```
user:password_hash
```

I kept the same format here when using `bcrypt`.

Reply:
Oooo because `bcrypt` is only creating a salted hash for the password right ?

Reply:
Yes it is

Reply:
Ok makes sense then :+1:

Reply:
So I tried it on `golem` and from what I see it seems to be working :)

Reply:
:tada: